### PR TITLE
Update: Move more less to scss

### PR DESCRIPTION
**Pull request by kevinhinterlong**
## Description
We're getting pretty close to removing our `less` styles and simplifying the use of our addons in consuming apps

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/core/app/styles/navi-core/components/navi-visualization-config/pie-chart.scss [hunk 1]
```diff
@@ -2,4 +2,5 @@
  * Copyright 2021, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
-@import 'table';
+
+// .pie-chart-config {}
```
Comment (jkusa) [inline packages/core/app/styles/navi-core/components/navi-visualization-config/pie-chart.scss:6]: Remove?

File: packages/core/app/styles/navi-core/components/navi-visualizations/table.scss [hunk 2]
```diff
@@ -3,34 +3,34 @@
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 
-@navi-table-max-wrap: 3;
-@navi-header-line-height: 1.3em;
+$navi-table-max-wrap: 3;
+$navi-header-line-height: 1.3em;
+$font-family-strong: 'HelveticaNeueW01-55Roma', Helvetica, Arial, sans-serif;
```
Comment (jkusa) [inline packages/core/app/styles/navi-core/components/navi-visualizations/table.scss:8]: Should we remove this?

File: packages/core/app/styles/navi-core/components/navi-visualizations/table.scss [hunk 3]
```diff
@@ -107,24 +108,24 @@
...
+        display: flex;
+        padding: $cell-padding;
 
         .metric {
           text-transform: capitalize;
         }
 
         &.sortable-item.is-dragging {
-          border: 1px solid @navi-lighter-gray;
+          background-color: map-get($denali-grey-colors, '100');
```
Comment (jkusa) [inline packages/core/app/styles/navi-core/components/navi-visualizations/table.scss:124]: should we use the background color here for theming?

File: packages/core/app/styles/navi-core/components/navi-visualizations/table.scss [hunk 4]
```diff
@@ -171,11 +172,11 @@
         }
 
         .table-header-cell {
-          background-color: @navi-white;
-          border-bottom: 2px @navi-lighter-gray solid;
+          background-color: map-get($denali-grey-colors, '100');
```
Comment (jkusa) [inline packages/core/app/styles/navi-core/components/navi-visualizations/table.scss:175]: Same here

**Review by jkusa — COMMENTED** [review]
Just small stuff

File: packages/core/app/styles/navi-core/components/navi-visualizations/table.scss (hunk 2 above)
Comment (kevinhinterlong) [reply packages/core/app/styles/navi-core/components/navi-visualizations/table.scss:8]: Keeping for now because it looks better than without

**Review by jkusa — APPROVED** [review]

**Pull request merged**